terraform-beyond-layer0/example-3/modules/guestbook_service/Dockerrun.aws.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Sorry, that statement was pretty vague. 

This kind of feeds of my earlier comment about showing users how to write good Terraform modules. Basically, in the 'real world' equivalent that this example is trying to show, the module for the `guestbook` service you are importing wouldn't exist in the same repo/directory tree. The module should be a "blueprint" for a service, and the configuration should be the real, actual resources created - and its very important we keep these things separate. 

This is something I'm not seeing in these examples, and this example is good candidate for it. By putting the "blueprint" module for the `guestbook` service as a subdir in this example, we are showing a pattern users shouldn't do